waku/waku_relay/protocol.nim

```diff
@@ -208,18 +223,38 @@ proc initRelayMetricObserver(w: WakuRelay) =
     for msg in msgs.messages:
       let (msg_id_short, topic, wakuMessage, msgSize) = decodeRpcMessageInfo(peer, msg).valueOr:
         continue
-      # message receive log happens in treaceHandler as this one is called before checks
+      # message receive log happens in onValidated observer as onRecv is called before checks
       updateMetrics(peer, topic, wakuMessage, msgSize, onRecv = true)
     discard
 
+  proc onValidated(peer: PubSubPeer, msg: Message, msgId: MessageId) =
+    let msg_id_short = shortLog(msgId)
+    let wakuMessage = WakuMessage.decode(msg.data).valueOr:
+      warn "onValidated: failed decoding to Waku Message",
+        my_peer_id = w.switch.peerInfo.peerId,
+        msg_id = msg_id_short,
+        from_peer_id = peer.peerId,
+        pubsub_topic = msg.topic,
+        error = $error
+      return
+
+    logMessageInfo(
+      w, shortLog(peer.peerId), msg.topic, msg_id_short, wakuMessage, onRecv = true
+    )
+
   proc onSend(peer: PubSubPeer, msgs: var RPCMsg) =
     for msg in msgs.messages:
       let (msg_id_short, topic, wakuMessage, msgSize) = decodeRpcMessageInfo(peer, msg).valueOr:
+        warn "onSend: failed decoding RPC info",
+          my_peer_id = w.switch.peerInfo.peerId, to_peer_id = peer.peerId
```

Comment:
not_idiomatic_no_? Here and elsewhere.

Reply:
No strong opinions, as it's mostly about how we log things, but I agree that would be nice to have everything unified in a single format.

On the other hand, DST already build its tools based on those keys, which are the ones we have been using for long.

@Ivansete-status opinions?

Reply:
@gabrielmer - it looks correct to my eyes
@SionoiS - do you mean the use of snake case? I think we tend to use that but if not the case I'm happy to unify

Reply:
I'll proceed and merge it this way so it enters the release. If we decide to change this, I'll open a separate PR for it :)